msal/build.gradle

```diff
@@ -90,8 +90,8 @@ android {
                 exclude 'com/microsoft/identity/client/e2e/tests/network'
             }
 
-            systemProperty 'robolectric.offline', 'true'
-            systemProperty 'robolectric.dependency.dir', './libs'
+            //systemProperty 'robolectric.offline', 'true'
```

Comment:
No longer needed?

Reply:
the problem is that whenever we change target version we need to download the new file and check it in the repo.....I'm thinking we can make it optional via cmd param....similar to how we've done a lot of other things...

Reply:
I can remove the comment.